keyboards/aeboards/constellation/config.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2018 Jason Williams (Wilba)
```

Comment:
Was this here by mistake (copy-paste) or ...???
Cause I don't see Wilba credited in the new files.

Reply:
Ah originally Wilba helped me create these files. Should I change this?

Reply:
I think something like this would be a nice/proper gesture in this case:
```
Copyright 2018 Jason Williams (Wilba)
Copyright 2021 Harrison Chan (Xelus)
```